coderd/database/dbpurge/dbpurge_test.go

```diff
@@ -183,20 +184,28 @@ func TestDeleteOldWorkspaceAgentLogs(t *testing.T) {
 		// given
 		agent := mustCreateAgentWithLogs(ctx, t, db, user, org, tmpl, tv, now.Add(-8*24*time.Hour), t.Name())
 
+		// Make sure that agent logs have been collected.
+		agentLogs, err := db.GetWorkspaceAgentLogsAfter(ctx, database.GetWorkspaceAgentLogsAfterParams{
+			AgentID: agent,
+		})
+		require.NoError(t, err)
+		require.NotZero(t, agentLogs, "agent logs must be present")
+
 		// when
 		closer := dbpurge.New(ctx, logger, db)
 		defer closer.Close()
 
 		// then
-		require.Eventually(t, func() bool {
-			agentLogs, err := db.GetWorkspaceAgentLogsAfter(ctx, database.GetWorkspaceAgentLogsAfterParams{
+		assert.Eventually(t, func() bool {
```

Comment:
```suggestion
		require.Eventually(t, func() bool {
```

Reply:
`require.Eventually` will fail with `Condition never met`. I want to run the follow-up assertion afterward to see what is on the list.